src/components/tables/cells/stats/Docs.tsx

```diff
@@ -1,20 +1,57 @@
-import { TableCell } from '@mui/material';
-import { tableBorderSx } from 'context/Theme';
-import { FormattedNumber } from 'react-intl';
+import { Box, TableCell, Tooltip, Typography } from '@mui/material';
+import {
+    semiTransparentBackgroundIntensified,
+    tableBorderSx,
+} from 'context/Theme';
+import { useMemo } from 'react';
+import { useIntl } from 'react-intl';
+import readable from 'readable-numbers';
 
 interface Props {
-    val?: number;
+    read?: boolean;
+    val?: number | null;
 }
 
-const Docs = ({ val }: Props) => {
+const Docs = ({ read, val }: Props) => {
+    const intl = useIntl();
+    const statsLoading = val === null;
+    const defaultedVal = val ?? 0;
+    const number = useMemo(
+        () => readable(defaultedVal, 2, false),
+        [defaultedVal]
+    );
+
     return (
         <TableCell
             sx={{
                 ...tableBorderSx,
+                minWidth: 'min-content',
                 maxWidth: 'min-content',
             }}
         >
-            <FormattedNumber value={val ?? 0} />
+            <Box sx={{ maxWidth: 'fit-content' }}>
+                <Tooltip
+                    title={`${defaultedVal} ${intl.formatMessage({
+                        id: read
+                            ? 'entityTable.stats.docs_read'
+                            : 'entityTable.stats.docs_written',
+                    })}`}
+                >
+                    <Typography
+                        sx={{
+                            transitionDelay: statsLoading ? '800ms' : '0ms',
+                            color: (theme) =>
+                                statsLoading
+                                    ? semiTransparentBackgroundIntensified[
+                                          theme.palette.mode
+                                      ]
+                                    : null,
+                        }}
+                    >
+                        {number} docs
```

Comment:
Any reason not to make this an `intl` string with a value?

Reply:
I just forgot to - I'll fix it.